src/nn/ops/lstm.ts

```diff
@@ -359,14 +359,16 @@ export class LstmCell extends Operation {
         const zero = tf.scalar(0);
         const starts = layout === MLLstmWeightLayout.iofg ?
         {i: 0, o: hiddenSize, f: 2 * hiddenSize, g: 3 * hiddenSize} :
-        /*ifog*/ {i: 0, f: hiddenSize, o: 2 * hiddenSize, g: 3 * hiddenSize};
+        /*ifgo*/ {i: 0, f: hiddenSize, g: 2 * hiddenSize, 0: 3 * hiddenSize};
```

Comment:
What does `0:` mean? I didn't know that key names were even allowed to start with digits :b (I figured they followed the same rules as other identifiers where they may contain digits but must start with a letter).

Reply:
> What does 0: mean?

It's a typo error, I've fixed it with last commit.